src/main/java/org/killbill/billing/plugin/adyen/api/mapping/klarna/PropertyMapper.java

```diff
@@ -236,4 +244,37 @@ public String toString() {
                    '}';
         }
     }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class LineAdjustment {
+        private Long amount;
+        private String type;
+        private String description;
+
+        public LineAdjustment() { }
+
+        public LineAdjustment(Long amount, String type, String description) {
+            this.amount = amount;
+            this.type = type;
+            this.description = description;
+        }
+
+        public Long getAmount() { return amount; }
+        public void setAmount(final Long amount) { this.amount = amount; }
+
+        public String getType() { return type; }
+        public void setType(final String type) { this.type = type; }
+
+        public String getDescription() { return description; }
+        public void setDescription(final String description) { this.description = description; }
+
+        @Override
+        public String toString() {
```

Comment:
Do we need to override equals and hashcode() methods?

Reply:
No. This is object is not used for equality check.